app/templates/partials/messages-status-en.html

```diff
@@ -34,32 +34,29 @@ <h3 class="heading-small">Delivered</h3>
 
 <h3 class="heading-small">Failed</h3>
 <p>
-  If a message fails to be delivered, you can retry after 72 hours or try a different channel. A message may fail to be delivered for several reasons:
+  If a message fails to be delivered, you can retry after 72 hours or try a different channel. If a message fails consistently you should remove these contacts from your database.
+</p>
+<p>
+  A message may fail to be delivered for several reasons:
 </p>
 
-<ul class="list-bullet">
+<ul style="list-style-type: disc; margin-left: 2em; margin-bottom: 20px;">
```

Comment:
```suggestion
<ul class="list list-bullet">
```
This what we use on the "Terms" page

https://github.com/cds-snc/notification-admin/blob/1375836399263888f7b3a687312558aea60fc71c/app/templates/views/terms-of-use.html#L19